src/coreComponents/mesh/FaceManager.cpp

```diff
@@ -96,7 +96,7 @@ void FaceManager::buildSets( NodeManager const & nodeManager )
   } );
 }
 
-void FaceManager::setDomainBoundaryObjects()
+void FaceManager::setDomainBoundaryObjects( ElementRegionManager const & elemRegionManager )
 {
   arrayView1d< integer > const isFaceOnDomainBoundary = getDomainBoundaryIndicator();
```

Comment:
this is actually the mpi boundary, right? We are flagging faces on both partition boundaries and domain boundaries.